nameserver/status.go

```diff
@@ -32,12 +33,15 @@ func NewStatus(ns *Nameserver, dnsServer *DNSServer) *Status {
 			entry.ContainerID,
 			entry.Addr.String(),
 			entry.Version,
-			entry.Tombstone})
+			entry.Tombstone,
+		})
 	}
 
 	return &Status{
 		dnsServer.domain,
+		dnsServer.upstream.Servers,
 		dnsServer.address,
 		dnsServer.ttl,
-		entryStatusSlice}
+		entryStatusSlice,
+	}
```

Comment:
Ok, I'll remove these changes (including the `import`s in `prog/weaver/http.go`)